### DISCUSSION TITLE
CoreClient & Web3Wallet stopped working without any code changes

### DISCUSSION THREAD

**Discussion by danylomakarenko** [WalletConnect/walletconnect-monorepo#4613] [Developer Support]
Project ID: f86076bc91705f6f753fd4a6802640a6
Platform: Android
Versions:
implementation(platform("com.walletconnect:android-bom:1.31.1"))
implementation("com.walletconnect:android-core")
implementation("com.walletconnect:web3wallet")

We released our [app](https://play.google.com/store/apps/details?id=com.app.renegade) and everything worked correctly for the first few hours.
Then without any code changes users stopped being able to connect to any dApp.
From the code side both CoreClient.initialize() and Web3Wallet.initialize() fails with the following error:
`Error(throwable=com.walletconnect.android.internal.common.exception.InvalidProjectIdException: Expected HTTP 101 response but was '403 Forbidden')`
The error above is received inside onError() callback for both CoreClient.initialize() and Web3Wallet.initialize()
The error message is not really helpful because I'm 100% sure that the project id is correct + as I mentioned it worked for the first few hours.

The following code is called inside Application.kt class
```
private fun setupWeb3Wallet() {
        val projectId = BuildConfig.WALLET_CONNECT_PROJECT_ID
        val serverUrl = "wss://relay.walletconnect.com?projectId=$projectId"
        val appMetaData = Core.Model.AppMetaData(
            name = "Renegade Wallet",
            description = "Renegade Wallet",
            url = "https://www.myrenegade.net",
            icons = listOf(),
            redirect = "renegade-web3wallet://request"
        )

        CoreClient.initialize(
            relayServerUrl = serverUrl,
            connectionType = ConnectionType.AUTOMATIC,
            application = this,
            metaData = appMetaData,
            onError = { error ->
                Timber.e("Error during CoreClient.initialize() : $error")
                scope.launch {
                    connectionStateFlow.emit(WalletConnectConnectionState.Error(error.throwable.message ?: ""))
                }
            }
        )

        Web3Wallet.initialize(Wallet.Params.Init(core = CoreClient)) { error ->
            Timber.e("Error during Web3Wallet.initialize() : $error")
        }
    }
```

**Comment by danylomakarenko** [comment]
@glitch-txs please have a look

**Comment by glitch-txs** [comment]
Could you upgrade to [the newest SDK version](https://github.com/WalletConnect/WalletConnectKotlinV2/releases/tag/BOM_1.32.1). Secondly, please check if your Cloud configuration is correct, SDK would only return a "Invalid Project Id" error if Relay returns it at the first place. Make sure it's not undefined, try to hard code it. Thanks!
